Packs/Imperva_WAF/Integrations/ImpervaWAF/ImpervaWAF.py

```diff
@@ -45,6 +45,74 @@ def login(self):
         extract_errors(res)
         self.session_id = res.get('session-id')
 
+    def get_ip_group_entities(self, group_name, table_name):
```

Comment:
`get` function that doesn't return anything? please update the name or return the triplet (hr, context, raw) to the calling function and call `return_outputs` there